conference.js

```diff
@@ -1853,7 +1875,7 @@ export default {
             APP.UI.onLocalRaiseHandChanged(raisedHand);
 
             this.isHandRaised = raisedHand;
-            // Advertise the updated status
+             // Advertise the updated status
```

Comment:
I had a thought and want your opinion: what if you had another method that did the force logic so the toggle functions could remain ridiculously simple? I foresee some drawbacks with added complexity due to there being another method floating around.